src/omneo/resources/profiles/index.ts

```diff
@@ -435,7 +436,7 @@ export default class Profiles extends Resource {
     })
   }
 
-  async getProfileDates (profileID: string): Promise<any> { // TODO Add Profile Dates Type
+  async getProfileDates (profileID: string): Promise<Array<ProfileDatesAttribute>> {
```

Comment:
Promise<ProfileDatesAttribute[]>
just to be consistent with the rest of the repo

Reply:
changed.